dds2val/src/ddsfeeder.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+
+#################################################################################
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License 2.0 which is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+import asyncio
+import contextlib
+import logging
+import os
+import signal
+import time
+from pathlib import Path
+
+# pylint: disable=unused-import
+import sensor_msgs  # # noqa: F401
+
+# F401 std_msgs module is used  to access the data classes
+# pylint: disable=unused-import
+import std_msgs  # # noqa: F401
+
+# F401 Vehicle module is used inside the eval block to access the data classes
+# pylint: disable=unused-import
+import Vehicle  # noqa: F401
+from cyclonedds.core import Listener
+from cyclonedds.domain import DomainParticipant
+from cyclonedds.sub import DataReader
+from cyclonedds.topic import Topic
+from grpc import ChannelConnectivity
+from kuksa_client.grpc import VSSClient, VSSClientError
+
+from . import databroker, vss2ddsmapper  # noqa: F401
```

Comment:
If changing this line to

`import databroker, vss2ddsmapper`

... then both my docker build and native run works